opensearch/src/main/java/org/opensearch/sql/opensearch/storage/scan/OpenSearchIndexScanQueryBuilder.java

```diff
@@ -39,8 +39,9 @@
 @VisibleForTesting
 class OpenSearchIndexScanQueryBuilder extends TableScanBuilder {
   @Override
-  public void pushDownOffset(int i) {
+  public boolean pushDownOffset(int i) {
```

Comment:
`pushDownOffset` was removed in #233
https://github.com/Bit-Quill/opensearch-project-sql/pull/233/files#diff-3334dc7821335d1c35cb35e3da00960106b6ac04c4ca092937c51c076056d272